packages/ui/tsconfig.json

```diff
@@ -5,25 +5,21 @@
     }
   },
   "compilerOptions": {
-    "target": "es5",
```

Comment:
Can you confirm that the only change here was outDir? Hard to read with the alphabetizing

Reply:
That's correct